src/pages/index.tsx

```diff
@@ -2,10 +2,12 @@ import { type NextPage } from "next";
 import Navbar from "../components/Navbar";
 import React from "react";
 import { useElementMouseRelativeAngle } from "../utils/hooks";
-import { signIn, signOut } from "next-auth/react";
+import { signIn} from "next-auth/react";
+// import { useRouter } from 'next/router';
 
 const Home: NextPage = () => {
   const { angle, ref } = useElementMouseRelativeAngle();
+  // const router = useRouter();
   return (
     <>
```

Comment:
I was thinking about resolving #24 right now by checking session and routerpush /dashboard